save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/DemoManagerController.kt

```diff
@@ -127,6 +140,10 @@ class DemoManagerController(
         .switchIfEmptyToNotFound {
             "Could not find project $projectName in organization $organizationName."
         }
+        .filter { projectPermissionEvaluator.hasPermission(authentication, it, Permission.DELETE) }
```

Comment:
why delete?

Reply:
I see it's part of requirements.
> READ is required for getting DemoStatus;
> WRITE is required for getting DemoInfo;
> DELETE is required for demo adding.

@sanyavertolet, why we want to have more strict permission for demo?

Reply:
In the issue, @sanyavertolet  wrote on which permissions to check

Reply:
To my mind, `DELETE` means deletion and creation. Demo is a kind of important part of a project so that only an owner can decide whether the project requires demo or not